tworaven_apps/eventdata_queries/urls.py

```diff
@@ -27,5 +27,23 @@
          views.api_upload_to_dataverse,
          name='api_upload_to_dataverse'
          ),
+    re_path(r'api/publish-dataset/get/(?P<dataset_id>[0-9]{1,10})$',
```

Comment:
Don't use dataset_id; pass the id of the `EventDataSavedQuery`
Even though it's not used now; it will be when checking if the user has access to publish that query--or to check if that file/query has already been published.

Reply:
don't need the `/get` in the url.  `api/publish-dataset/(?P<dataset_id>[0-9]{1,10})$` is fine

Reply:
I'll make this change and push the code